specification/communication/resource-manager/Microsoft.Communication/stable/2023-03-31/CommunicationServices.json

```diff
@@ -0,0 +1,840 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Services Management API",
+    "version": "2023-03-31",
+    "description": "REST API for Azure Communication Services"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Communication/operations": {
+      "get": {
+        "description": "Lists all of the available REST API operations of the Microsoft.Communication provider.",
+        "operationId": "Operations_List",
+        "summary": "List Operations",
+        "tags": [
+          "Operations"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response describes the list of operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/communicationServices/operationsList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Communication/checkNameAvailability": {
+      "post": {
+        "description": "Checks that the CommunicationService name is valid and is not already in use.",
+        "operationId": "CommunicationServices_CheckNameAvailability",
```

Comment:
@matthohn-msft Why do you change `CommunicationService` to `CommunicationServices` compared with last stable api-version https://github.com/Azure/azure-rest-api-specs/blob/bbee55edb05169aba6d8d1944c0e2bc2b9408943/specification/communication/resource-manager/Microsoft.Communication/stable/2020-08-20/CommunicationService.json#L68

Reply:
CommunicationServices was made plural because there are two service types for Microsoft.Communication. We are adding a new communication service type with this change, EmailServices.  Mike Kistler has approved this breaking change.